snippets/language-ruby.cson

```diff
@@ -15,6 +15,9 @@
   'case … end':
     'prefix': 'case'
     'body': 'case ${1:object}\nwhen ${2:condition}\n\t$0\nend'
+  'begin … rescue … end':
+    'prefix': 'bre'
```

Comment:
That actually isn't consistent with the existing Ruby snippets. See `if ... else ... end` using `ife`.